### DISCUSSION TITLE
IPV6端口无法放行

### DISCUSSION THREAD

**Discussion by jihanc** [immortalwrt/immortalwrt#1278] [Q&A]
### Describe the bug

无法放行IPV6特定端口流量。我确定我所开放的端口没有被拦截

### ImmortalWrt version

r27625-416c8c5c91

### ImmortalWrt release

23.05.2

### ImmortalWrt target/subtarget

mediatek/filogic

### Device

H3C Magic NX30 Pro (NMBM layout)

### Image kind

Official downloaded image

### Steps to reproduce

防火墙配置如图，无法放行IPV6流量。
![图片](https://github.com/immortalwrt/immortalwrt/assets/78247232/bbbfeefe-4ac2-4d92-8481-4c1d0ba4bd2f)

将这里的转发改成接受就没有问题，也无须上述配置。
![图片](https://github.com/immortalwrt/immortalwrt/assets/78247232/d0fda378-b8c7-4be4-9b8b-36ab9ee9a899)

### Expected behaviour

不知道是什么原因导致 防火墙-通信规则 没有正常生效，直接转发暴露太多端口不安全。

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
https://www.cnblogs.com/osnosn/p/11781359.html

**Reply by jihanc** [reply]
我之前已经阅读过这一篇文章，问题仍然存在
![图片](https://github.com/immortalwrt/immortalwrt/assets/78247232/8a7ea5ee-26b5-41a2-b247-bf7c2169dc91)
![图片](https://github.com/immortalwrt/immortalwrt/assets/78247232/a11b51b6-88e6-4667-89e1-22c6cf441bb4)

**Reply by jihanc** [reply]
重新按照文章设置，问题仍未解决
![图片](https://github.com/immortalwrt/immortalwrt/assets/78247232/f9c91590-1628-4315-a0e5-4e9f8a204488)

**Reply by jihanc** [reply]
这是打开默认转发的情况

![无标题](https://github.com/immortalwrt/immortalwrt/assets/78247232/dd7a7d0c-8816-4bf0-ab5e-15f483600c6f)
